telemetry/vscode/src/generate.ts

```diff
@@ -127,8 +139,49 @@ export function generateTelemetry(telemetryJson: MetricDefinitionRoot): string {
     return str
 }
 
+export async function generate(args: CommandLineArguments) {
+    let output = `
+    /*!
+     * Copyright ${new Date().getUTCFullYear()} Amazon.com, Inc. or its affiliates. All Rights Reserved.
+     * SPDX-License-Identifier: Apache-2.0
+     */
+
+    import { ext } from '../extensionGlobals'
+    `
+
+    const rawDefinitions: MetricDefinitionRoot = args.inputFiles
+        .map(path => {
+            const fileInput = readFileSync(path, 'utf8')
+            return validateInput(fileInput)
+        })
+        .reduce(
+            (item: MetricDefinitionRoot, input: MetricDefinitionRoot) => {
+                item.types!.push(...(input.types ?? []))
+                item.metrics.push(...input.metrics)
+                return item
+            },
+            { types: [], metrics: [] }
+        )
+    // Allow read in files to overwrite default definitions. First one wins, so the extra
```

Comment:
does "read in" mean "discovered" 
```suggestion
    // Allow discovered files to overwrite default definitions. First one wins, so the extra
```

Reply:
There's no discovery happening? The files that are specified explicitly to be part of the generate